src/main/java/edu/hw3/Task7/TreeMapComparator.java

```diff
@@ -0,0 +1,18 @@
+package edu.hw3.Task7;
+
+import java.util.Comparator;
+
+public class TreeMapComparator<T> implements Comparator<T> {
+    @Override
+    public int compare(T o1, T o2) {
+        if (o1 == null && o2 == null) {
+            return 0;
+        } else if (o1 == null) {
```

Comment:
`Enhacement`давай не будем писать else if после return 
```suggestion
        }
         if (o1 == null) {
```